src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -212,21 +213,33 @@ class ObjectField extends React.Component {
       </div>
     );
 
+    // Id's are not always unique on the review and submit page
+    const id =
+      isRoot && formContext.onReviewPage
+        ? Object.keys(idSchema)
+            .reduce((ids, key) => {
+              ids.push(idSchema[key].$id || '');
+              return ids;
+            }, [])
+            .filter(k => k)
+            .join('_')
+        : idSchema.$id;
```

Comment:
I think I'm not following this. :thinking: 
How is this different than what's in `idSchema.$id`? How does it ensure unique IDs?

Reply:
When there are two instances of component, then both will have a similar `idSchema`. 

I've included an example in the unit test:

```js
const idSchema1 = {
  $id: 'root',
  test: {
    $id: 'root_test',
  },
};
const idSchema2 = {
  $id: 'root',
  test2: {
    $id: 'root_test2',
  },
};
```

without pulling in the `$id` from additional `$id`s, then the form element id won't be unique. Both components will render with an id of `root__title`. The function appends the additional id, so from the example id, we'll end up with ids of `root_test__title` and `root_test2__title` respectively.

In the actual 526 form, we have the `selectArrayItemsWidget` render a card on both the `ratedDisabilities` and `unemployabilityDisabilities` page. When on the review and submit page, both can potentially be rendered when the accordion is open and _both_ are in edit mode. Maybe because of this situation, this is an edge case?

Reply:
Interesting. I think the fundamental assumption here is that a field is only used in one place in the entire form. When that's not the case, I think that's when this gets funky.

This seems pretty safe, though, since it's just on the review page. If it were on a normal form page, we'd probably run into some trouble with the automated form tester.